frontend/models/sections/banners-section.ts

```diff
@@ -0,0 +1,33 @@
+import { filterFalsyItems } from '@helpers/application-helpers';
+import type { BannersSectionFieldsFragment } from '@lib/strapi-sdk';
+import {
+   bannerFromStrapiFragment,
+   BannerSchema,
+} from '@models/components/banner';
+import { z } from 'zod';
+
+export type BannersSection = z.infer<typeof BannersSectionSchema>;
+
+export const BannersSectionSchema = z.object({
+   type: z.literal('Banners'),
+   id: z.string(),
+   banners: z.array(BannerSchema),
+});
+
+export function bannersSectionFromStrapi(
+   fragment: BannersSectionFieldsFragment | null | undefined,
+   sectionId: string
+): BannersSection | null {
+   const strapiBanners = fragment?.banners?.data ?? [];
+   const banners = filterFalsyItems(
+      strapiBanners.map(bannerFromStrapiFragment)
+   );
+
+   if (banners == null) return null;
```

Comment:
When could `banners == null` ever be true? It seems like `filterFalsyItems` always returns an array, and `[] == null` is false in JS.